### DISCUSSION TITLE
Great Job

### DISCUSSION THREAD

**Discussion by lbarreiro** [ksheumaker/homeassistant-apsystems_ecur#56] [General]
Hi,
Just need to say thank's for all the work and time you spend.
Work flawless and so far no bug's 

Keep up the good work
Thank's
LBarreiro

**Comment by tediroca** [comment]
Hi,

Just got my DS3 microinverters installed with an ECU-C, and the integration has been working nicely for a couple of days. 

I just want to say thank you, and would be keen to help with anything than can support the author. I'm not a skilled dev, but can test things and research as needed :)

Cheers.

**Reply by jhdebruin** [reply]
Hi, Fulle aggree, just installed with an ECU-R , had to made some steps with enabling Wifi and finding the ECU on my network, but after that all worked like a rocket !
